animations/pubspec.lock

```diff
@@ -158,3 +158,4 @@ packages:
     version: "2.0.8"
 sdks:
   dart: ">=2.9.0-14.0.dev <3.0.0"
+  
```

Comment:
```suggestion

```

Still showing the diff. For next time to avoid pushing the `pubspec.lock` changes if you are working on a lower version, you can do `git add lib/` to only add the changes of the lib folder to git. You will need to remove white space. There are currently two white spaces on line 161. That's why it's still showing the difference.